ci/utils.groovy

```diff
@@ -1,11 +1,7 @@
 nix = load 'ci/nix.groovy'
 
 def getVersion(type = null) {
```

Comment:
Then I guess we should drop the `type` argument and remove it from all the `getVersion()` calls.

Reply:
You're right, I missed that, fixing.